### PR TITLE
Allow CPObjectController to get more attributes from the xib/cib file.

### PR DESCRIPTION
**Pull request by mrcarlberg**
This pull request reads more attributes from the xib/cib file. The attributes are entity name and uses lazy fetching.

### CODE REVIEW

File: AppKit/CPObjectController.j [hunk 1]
```diff
@@ -374,12 +424,17 @@ var CPObjectControllerContentKey                        = @"CPObjectControllerCo
...
+            content = [aCoder decodeObjectForKey:CPObjectControllerContentKey];
 
         [self setObjectClass:objectClass || [CPMutableDictionary class]];
         [self setEditable:[aCoder decodeBoolForKey:CPObjectControllerIsEditableKey]];
         [self setAutomaticallyPreparesContent:[aCoder decodeBoolForKey:CPObjectControllerAutomaticallyPreparesContentKey]];
-        [self setContent:[aCoder decodeObjectForKey:CPObjectControllerContentKey]];
+        [self setUsesLazyFetching:[aCoder decodeBoolForKey:CPObjectControllerUsesLazyFetchingKey]];
+        _isUsingManagedProxy = [aCoder decodeBoolForKey:CPObjectControllerIsUsingManagedProxyKey];
+        _managedProxy = [aCoder decodeObjectForKey:CPObjectControllerManagedProxyKey];
+        if (content != nil)
```
Comment (Dogild) [inline AppKit/CPObjectController.j]: Style here, add a line breaks

File: AppKit/CPObjectController.j [hunk 2]
```diff
@@ -398,6 +453,10 @@ var CPObjectControllerContentKey                        = @"CPObjectControllerCo
 
     [aCoder encodeBool:[self isEditable] forKey:CPObjectControllerIsEditableKey];
     [aCoder encodeBool:[self automaticallyPreparesContent] forKey:CPObjectControllerAutomaticallyPreparesContentKey];
+    [aCoder encodeBool:[self usesLazyFetching] forKey:CPObjectControllerUsesLazyFetchingKey];
+    [aCoder encodeBool:_isUsingManagedProxy forKey:CPObjectControllerIsUsingManagedProxyKey];
+    if (_managedProxy)
```
Comment (Dogild) [inline AppKit/CPObjectController.j]: Same as above

**Comment by mrcarlberg** [comment]
I have now fixed the formatting

**Comment by Dogild** [comment]
Thanks :)

**Pull request merged**